wp-content/civi-extensions/goonjcustom/CRM/Goonjcustom/state_codes.php

```diff
@@ -0,0 +1,39 @@
+<?php
+return [
+	'Andaman and Nicobar Islands' => 'ANN',
```

Comment:
Let's use CiviCRM `abbreviation` as the array keys instead of state `name`. The chances of changes to abbreviation is less than compared to chances of change in the name.
```suggestion
	'AN' => 'ANN',
```